pkg/operator/ceph/controller/cluster_info.go

```diff
@@ -371,23 +372,6 @@ func createClusterAccessSecret(clientset kubernetes.Interface, namespace string,
 	return nil
 }
 
-// ParseMonEndpoints parses a flattened representation of mons and endpoints in the form
-// <mon-name>=<mon-endpoint> and returns a list of Ceph mon configs.
-func ParseMonEndpoints(input string) map[string]*cephclient.MonInfo {
```

Comment:
Can we keep this method and delete the one in mon.go? The mon package should be able to reference this `controller` package without a circular reference. This `controller` package was created for similar methods to avoid circular references.

Reply:
Ok sure. I'm thinking of moving the `endpoint.go` file to `pkg/operator/ceph/controller`. This would move `ParseMonEndpoints` to the `controller` package as well as `FlattenMonEndpoints`. `FlattenMonEndpoints` is only referenced in the `mon` package.

Does this sound ok?

Reply:
If FlattenMonEndpoints is only needed in the `mon` package, then better to leave it in that package and change it to `flattenMonEndpoints` so it's only for the package.

Why create endpoints.go? We can just revert the change to this cluster_info.go, right?

Reply:
I thought the functions might stay together because they're related, but what you've said makes sense. I'll revert the change to `cluster_info.go`.

Reply:
I've made the changes.